pom.xml

```diff
@@ -164,7 +164,7 @@
         <dependency>
             <groupId>uk.gov.ons.ctp.integration.common</groupId>
             <artifactId>census-int-case-api-client</artifactId>
-            <version>0.0.7</version>
+            <version>0.0.8-SNAPSHOT</version>
```

Comment:
It shouldn't be using a snapshot release.

Reply:
Agreed. It is waiting approval of this [case client api pull request](https://github.com/ONSdigital/census-int-case-api-client/pull/7). It is a facet of our continuous integration build I believe. The version will move to 0.0.8 when the case client api pull request is approved and I will be able to update this dependency in the POM here by removing the "-SNAPSHOT". 
Maybe a bit premature in raising this pull request, the other should be merged first then I can change this to the final version before this is merged. One simple little change I didn't wait for but maybe I shouldn't have raised this pull request quite yet.

Reply:
Done. Dependency updated to final 0.0.8 version now associated pull request approved.